src/components/tabs/VtxTab.vue

```
@@ -0,0 +1,874 @@
+        onMounted(async () => {
+            await loadVtxConfig();
+
+            // Start device status polling
+            addLocalInterval("vtx_device_status_pull", updateDeviceStatus, 1000);
+
+            i18n.localizePage();
+            GUI.content_ready();
+        });
+
+        onUnmounted(() => {
+            localIntervals.forEach((name) => GUI.interval_remove(name));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)